src/components/InlinePopover/layouts/TwoButton.js

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import { css } from '@emotion/react';
+import styled from '@emotion/styled';
+import { capitalize } from 'lodash';
+
+import {
+  Button,
+  Icon,
+  Popover,
+  useInstrumentedHandler,
+} from '@newrelic/gatsby-theme-newrelic';
+
+const TwoButton = ({
+  text,
+  header,
+  learnMoreUrl,
+  primaryButton,
+  primaryButtonUrl,
+  secondaryButton,
+  secondaryButtonUrl,
+  type,
+  ...props
+}) => {
+  const capsType = capitalize(type);
+  const instrumentation = (eventName) => ({
+    category: `${capsType}Popover`,
```

Comment:
Nit: In `InlinePopover.js` we are sending `${type}InlinePopover` Is there a reason these should be different, or is it just an oversight?

Reply:
ah good catch, no they should be the same

Reply:
changed this in `InlinePopover.js` to just `${type}Popover`